frontend/src/components/worksheets/ExtraWorksheetHTML/ExtraWorksheetHTML.js

```diff
@@ -121,7 +133,7 @@ const GlossaryModal = ({ show, toggle }) => (
                     </tr>
                     <tr>
                         <td>
-                            <kbd>c</kbd>
+                            <kbd>c c</kbd>
```

Comment:
We should probably mention in the PR body and release notes that this keyboard shortcut has changed.

cc @percyliang